tools/free_energy/alchemical_run/gmx_fep.xml

```diff
@@ -0,0 +1,518 @@
+               <param name="coul"   type="text" value="0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.00 0.000 0.00 0.11 0.22 0.33 0.44 0.56 0.67 0.78 0.89 1.00 " label="Coulomb Lambdas (scaling electrostatics)"/>
+               <param name="vdw"    type="text" value="0.00 0.04 0.08 0.11 0.15 0.19 0.23 0.27 0.30 0.34 0.38 0.42 0.46 0.49 0.53 0.57 0.61 0.65 0.68 0.72 0.76 0.80 0.84 0.87 0.91 0.95 0.97 0.98 0.99 0.999 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00" label="Van der Waals Lambdas (scaling vdw interactions)"/>
+               <param name="bonded" type="text" value="0.00 0.04 0.08 0.11 0.15 0.19 0.23 0.27 0.30 0.34 0.38 0.42 0.46 0.49 0.53 0.57 0.61 0.65 0.68 0.72 0.76 0.80 0.84 0.87 0.91 0.95 0.97 0.98 0.99 0.999 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00" label="Bonded Lambdas (scaling torsion)"/> 
+            </when>
+        </conditional>
+        <conditional name="feoptions">
+            <param name="feop" type="select" label="Free energy variables">
+                <option value="default">Use default options</option>
+                <option value="modify">Modify default options</option>
+            </param>
+            <when value="default"/>
+            <when value="modify">
+            <param name="scalpha" type="float" value="0.5" label="The soft-core alpha parameter (sc-alpha)" />
+            <param name="scrpower" type="integer" value="6" label="The power of the radial term in the soft-core equation (sc-r-power)" />
+            <param name="scpower" type="integer" value="1" label="The power for lambda in the soft-core function (sc-power)" />
+            <param name="sccoul" type="text" value="no" label="Apply the soft-core free energy interaction transformation to the Columbic interaction of a molecule (sc-coul)? "/>
+            <param name="scsigma" type="float" value="0.3" label="The soft-core sigma" />
+            <param name="couplelambda0" type="text" value="vdw-q" label="Interactions at lambda=0" help="options are vdw-q, vdw, q, or none"/>
+            <param name="couplelambda1" type="text" value="none" label="Interactions at lambda=1" help="options are vdw-q, vdw, q, or none"/>
+            <param name="coupleintramol" type="text" value="no" label="Turn off intra-molecular interactions?" />
+            </when>
+        </conditional>
+        <param name="temperature" type="float" value="300.0" label="Temperature /K" />
+        <param name="pressure" type="float" value="1.0" label="Pressure /bar" />
+        <param name="input_fep" type="select" label="Generate inputs files only?">
+            <option value="no">No</option>
+            <option value="yes">Yes</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="fepinpout" format="tar" from_work_dir="input_files.tar" label="Input files"/>
```

Comment:
have a look at https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-filter, I think you can add a filter to all these outputs

Reply:
ping @tsenapathi; this will let you customise which outputs the user gets depending on what they choose for `input_fep`